### DISCUSSION TITLE
نصب نسخه جدید مرزبان

### DISCUSSION THREAD

**Discussion by moonjom** [Gozargah/Marzban#304] [General]
برای نصب مرزبان نسخه جدید مراحل زیر را انجام دهید . 

مرحله اول : آپدیت لینوکس

`apt update -y&&apt upgrade -y`

مرحله دوم : نصب راحت به وسیله اسکریپت زیر 

`sudo bash -c "$(curl -sL https://github.com/Gozargah/Marzban-scripts/raw/master/marzban.sh)" @ install
`

تبریک . نصب پنل تمام شد , حالا با زدن ctrl+c از لاگ های مرزبان خارج می شویم.

مرحله سوم : حالا با زدن دستور زیر یک ادمین روت برای ورود به پنل تعریف می کنیم .

`marzban cli admin create --sudo
`

حالا با وارد کردن آدرس زیر میتوانید  وارد پنل کاربری خود بشید .

`http://YOUR_IP_SERVER:8000/dashboard/login`

مرحله چهارم : برای گرفتن سرتیفیکت مراحل زیر را دنبال کنید : 

`sudo apt install -y socat cron`

`curl https://get.acme.sh | sh -s email=YOUR_EMAIL 
`
`mkdir -p /var/lib/marzban/certs/ `

```
~/.acme.sh/acme.sh --issue --standalone -d YOUR_DOMAIN \
--key-file /var/lib/marzban/certs/key.pem \
--fullchain-file /var/lib/marzban/certs/fullchain.pem
```
بعد از اجرای کدهای بالا , وارد فایل .env در پوشه /opt/marzban می شویم و دو خط uvicorn رو از کامند در میاریم و exmple.com رو نیز حذف می کنیم.

![Screenshot from 2023-06-02 12-22-16](https://github.com/Gozargah/Marzban/assets/122884265/60d9ec36-58b3-464c-95db-b6ed8f2473a1)

تمام ! حالا فایل رو سیو کنید و با زدن کدهای زیر داکر رو ریست بدید .

`cd /opt/marzban/
`
`docker compose down
`
`docker compose up -d`

و سرانجام پنل خود رو میتونید با دامنه با آدرس زیر بالا بیارید :
https://YOUR_DOMAIN:8000/dashboard/

نکته :  اگر میخواهید با پروکسی روشن کلادفلر پنل رو بالا بیارید دقت کنید پورت پنل رو باید حتما یکی از پورت های کلادفلر بزارید .

**Comment by hesampn** [comment]
سلام خسته نباشید من پنل روی سرور نصب میکنم این خطلا به من میده مشکل از کجا هست اگر امکانش هست راهنمایی کنید


Error response from daemon: AppArmor enabled on system but the docker-default profile could not be loaded: running `/sbin/apparmor_parser apparmor_parser -Kr /var/lib/docker/tmp/docker-default1396702888` failed with output: apparmor_parser: Unable to replace "docker-default".  Permission denied; attempted to load a profile while confined?

**Comment by nilofari700** [comment]
سلام من قبلا یوز و پس ادمین زده بودم ی کلمه هر دو مشابه . الان اینو بخام حذف کنم چه کنم؟ چون با دستور ادمین جدید تعریف کردم و وارد پنل میشم ولی با قبلی هم میشه وارد شد. ممنون

**Comment by Mjzaret** [comment]
چطور میتونیم نود رو به ورژن جدید مرزبان اضافه کنیم؟

**Reply by abedin70** [reply]
https://github.com/NaliTeam/AutoNode/blob/main/README-fa.md

اینو تست کن

**Comment by xRay-v2Ray** [comment]
نه اوکیه
مشکلی پیش نمیاد

**Comment by Moriartieee** [comment]
پنل رو چطور میشه به کامل حذف کرد؟

**Reply by RenasPasha** [reply]
marzban uninstall 
بعد هرچی گزینه اومد y بزن و بعد اینتر

**Comment by Morteza-DM** [comment]
درود نصب داکر با این پنل انجام میشه ؟ دستور نصبش با داکر رو میفرستید ؟

**Comment by Aa1122811228** [comment]
سلام چطور نصبش کمم

**Reply by RenasPasha** [reply]
اگه رو سرورت از قبل مرزبان داری دستور اپدیت (marzban update) رو بزن
بعد فایل docker-compose رو چک کن  خط سوم یا چهارم باید کلمه latest باشه
اگ میخوای تازه مرزبان رو سرورت نصب کنی همون دستور نصب و بزنی اخرین نسخه رو نصب میکنه برات

**Comment by xalilhajani-dot** [comment]
کد باید تو ssh بزنم یا اپtermius?

**Comment by mortezamasjedi121-pump** [comment]
> برای نصب مرزبان نسخه جدید مراحل زیر را انجام دهید . 
> 
> مرحله اول : آپدیت لینوکس
> 
> `apt update -y&&apt upgrade -y`
> 
> مرحله دوم : نصب راحت به وسیله اسکریپت زیر 
> 
> `sudo bash -c "$(curl -sL https://github.com/Gozargah/Marzban-scripts/raw/master/marzban.sh)" @ install
> `
> 
> تبریک . نصب پنل تمام شد , حالا با زدن ctrl+c از لاگ های مرزبان خارج می شویم.
> 
> مرحله سوم : حالا با زدن دستور زیر یک ادمین روت برای ورود به پنل تعریف می کنیم .
> 
> `marzban cli admin create --sudo
> `
> 
> حالا با وارد کردن آدرس زیر میتوانید  وارد پنل کاربری خود بشید .
> 
> 
> `http://YOUR_IP_SERVER:8000/dashboard/login`
> 
> 
> مرحله چهارم : برای گرفتن سرتیفیکت مراحل زیر را دنبال کنید : 
> 
> `sudo apt install -y socat cron`
> 
> `curl https://get.acme.sh | sh -s email=YOUR_EMAIL 
> `
> `mkdir -p /var/lib/marzban/certs/ `
> 
> ```
> ~/.acme.sh/acme.sh --issue --standalone -d YOUR_DOMAIN \
> --key-file /var/lib/marzban/certs/key.pem \
> --fullchain-file /var/lib/marzban/certs/fullchain.pem
> ```
> بعد از اجرای کدهای بالا , وارد فایل .env در پوشه /opt/marzban می شویم و دو خط uvicorn رو از کامند در میاریم و exmple.com رو نیز حذف می کنیم.
> 
> ![Screenshot from 2023-06-02 12-22-16](https://github.com/Gozargah/Marzban/assets/122884265/60d9ec36-58b3-464c-95db-b6ed8f2473a1)
> 
> تمام ! حالا فایل رو سیو کنید و با زدن کدهای زیر داکر رو ریست بدید .
> 
> `cd /opt/marzban/
> `
> `docker compose down
> `
> `docker compose up -d`
> 
> و سرانجام پنل خود رو میتونید با دامنه با آدرس زیر بالا بیارید :
> https://YOUR_DOMAIN:8000/dashboard/
> 
> نکته :  اگر میخواهید با پروکسی روشن کلادفلر پنل رو بالا بیارید دقت کنید پورت پنل رو باید حتما یکی از پورت های کلادفلر بزارید .
>

**Comment by aymanwk44-ops** [comment]
net
